README.rst

```diff
@@ -0,0 +1,31 @@
+cpython-source-deps
+===================
+
+Source for packages that the CPython build process depends on.
+
+It is currently expected that this will only be useful on Windows,
+and in any case you should never need to clone this repository
+unless you are updating its contents.
+
+.. contents::
+
+Updating SQLite Source Dependencies
```

Comment:
I think we can generalize this.  Most dependencies (sqlite, bzip2, xz, zlib) follow the same instructions; tcl and tk also follow the same instructions, but have an extra step afterwards that @zooba has to do (updating [cpython-bin-deps](../../../cpython-bin-deps/)); openssl will have a much-expanded step 5.  Maybe leave the steps below as is, but note that `sqlite` is just used an an example, and the same steps apply to the ones I listed above?

Reply:
Thanks :) so I tried my best at explaining that the other dependencies can be updated in similar fashion. Hope the latest change makes sense..